### PR TITLE
.coafiles: Add jinja2bear

### PR DESCRIPTION
**Pull request by AkshJain99**
jinja2bear is added for templates.

Closes https://github.com/coala/community/issues/129

### CODE REVIEW

File: templates/contributors.html [hunk 1]
```diff
@@ -26,7 +26,7 @@ <h1>Details of all the contributors</h1>
                 <p>teams:
                 {% for team in contributor.teams.all %}
                     {{ team.name }}
-                {% endfor %}
+                {% endfor %}{# for team in contributor.teams.all #}
```
Comment (Naveenaidu) [inline templates/contributors.html]: Shouldn't need to be necessary once you made the `check_end_label = False`.

Comment (AkshJain99) [reply templates/contributors.html]: but gitmate is throwing error if I do not, do these changes inspite of ```0check_end_label = False```

File: templates/openhub.html [hunk 2]
```diff
@@ -38,6 +38,5 @@ <h1>All of our Portfolio Projects</h1>
       </div>
     </div>
     <hr>
-    {% endfor %}{# for project in projects #}
-  </body>
+    {% endfor %}{# for project in projects #} {# for project in projects #}  </body>
```
Comment (Naveenaidu) [inline templates/openhub.html]: `{# for project in projects #} ` have been repeated twice. This does not seem right. Please look into it.

Comment (AkshJain99) [reply templates/openhub.html]: actually earlier i wrote it once but then gitmate was showing this error 
```
 gitmate-bot 15 hours ago Member

Unlabeled control end tag

Origin: Jinja2Bear, Section: all.jinja.

The issue can be fixed by applying the following patch:

--- a/tmp/tmptzzzojzz/templates/openhub.html
+++ b/tmp/tmptzzzojzz/templates/openhub.html
@@ -38,6 +38,6 @@
       </div>
     </div>
     <hr>
-    {% endfor %} {# for project in projects #}
+    {% endfor %}{# for project in projects #} {# for project in projects #}
   </body>
 </html>

```

File: templates/contributors.html (hunk 1 above)
Comment (AkshJain99) [reply templates/contributors.html]: I think ```check_end_lables = False``` is not working and why it is not working , is it the problem of bear version maybe it release with new release

Comment (AkshJain99) [reply templates/contributors.html]: okay thanks @Naveenaidu

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by AkshJain99** [comment]
is this pr merged @jayvdb ?

**Comment by jayvdb** [comment]
ack https://github.com/coala/community/pull/229/commits/569c3ccabb58514aca82b419b31511f494bfce46

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**